components/atoms/ActionButton.js

```diff
@@ -0,0 +1,192 @@
+import PropTypes from 'prop-types'
```

Comment:
This button isn't being used in this PR

Reply:
it was on dc-34 but not needed with latest changes for the splash page

Reply:
removed